app/models/cloud_volume.rb

```diff
@@ -18,6 +19,14 @@ class CloudVolume < ApplicationRecord
 
   acts_as_miq_taggable
 
+  supports :cloud_volume_backup_create do
+    unsupported_reason_add(:cloud_volume_backup_create, _("Backup unsupported")) unless validate_backup_create
```

Comment:
@durandom added note - I think it makes sense to leave supports on the main cloud_volume class, as that seems to be the common design pattern that makes it easy for other providers to eventually add support

I think I copied this code pattern from models/service.rb.  Just let me know if there's a different one I should follow!